library/src/components/Tags.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Tag as TagType } from '@asyncapi/parser';
+
+interface Props {
+  tags?: TagType[];
+}
+
+export const Tags: React.FunctionComponent<Props> = ({ tags }) => {
+  if (!tags || !tags.length) {
+    return null;
+  }
+
+  return (
+    <div className="mt-4">
+      {tags.map(tag => (
+        <Tag tag={tag} key={tag.name()} />
+      ))}
+    </div>
+  );
+};
+
+const Tag: React.FunctionComponent<{ tag: TagType }> = ({ tag }) => {
```

Comment:
I usually expect a component per file. Also, this component is not exported so I'd not be able to render it alone in Storybook. I think you should move this component to another file.

Reply:
I decided that rendering a single tag is pointless and treated it as a sub-component of the Tags component, but I will move it to separate file for you 😘

Reply:
hahaha! it's not pointless. It is pointless in your context but may not be when we present the design sytem.